galleon-pack/src/main/resources/layers/standalone/observability/layer-spec.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" ?>
 <layer-spec xmlns="urn:jboss:galleon:layer-spec:1.0" name="observability">
     <dependencies>
-        <layer name="microprofile-config"/>
-        <layer name="microprofile-metrics"/>
+        <layer name="microprofile-config" optional="true"/>
```

Comment:
Further to my last point; this layer has the health subsystem as a fixed feature group, not an excludable layer and health requires config.

Reply:
I missed, that I added a comment and removed the optional attribute.